src/components/ConversionServerErrorModal/index.tsx

```diff
@@ -13,26 +13,27 @@ interface ConversionServerErrorModalProps {
 const ConversionServerErrorModal: React.FC<ConversionServerErrorModalProps> = ({
     closeModal,
 }) => {
+    const footerButtons = (
+        <Button className="primary-button" onClick={closeModal}>
+            OK
+        </Button>
+    );
+
     return (
         <CustomModal
             className={styles.serverErrorModal}
-            title="File import cannot be completed"
-            open
-            footer={
-                <Button type="primary" onClick={closeModal}>
-                    OK
-                </Button>
-            }
-            centered
-            onCancel={closeModal}
+            titleText="File import cannot be completed"
+            footerButtons={footerButtons}
+            closeHandler={closeModal}
         >
             <p className={styles.warningText}>
-                {Exclamation} We&apos;re sorry, the server is currently
-                experiencing an issue.
+                {Exclamation}{" "}
+                {` We're sorry, the server is currently
```

Comment:
nit, but it seems like you have double spaces here? I would use just one method for creating the space